data/FilterList.json

```diff
@@ -965,7 +965,9 @@
     "publishedDate": "2018-05-23T21:53:55",
     "syntaxId": 4,
     "updatedDate": "2018-10-09T01:28:00",
-    "viewUrl": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt"
+    "viewUrl": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt",
+    "viewUrlMirror1": "https://raw.githubusercontent.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt",
+    "viewUrlMirror2": "https://cdn.rawgit.com/NanoAdblocker/NanoFilters/master/NanoMirror/NanoDefender.txt"
```

Comment:
```suggestion
```

Reply:
Not sure if the beta GitHub requested change feature works properly. But, essentially, let's not duplicate the same url in both viewUrl and viewUrlMirror1.